helm/botkube/values.yaml

```diff
@@ -46,11 +46,25 @@ rbac:
   # the default in-cluster K8s client configuration. Instead, they should always use kubeconfig specified directly for a given plugin.
   serviceAccountMountPath: /var/run/7e7fd2f5-b15d-4803-bc52-f54fba357e76/secrets/kubernetes.io/serviceaccount
   create: true
-  rules:
-    - apiGroups: ["*"]
-      resources: ["*"]
-      verbs: ["get", "watch", "list"]
-  staticGroupName: &static-group-name "botkube-plugins-default"
+  # -- Deprecated. Use `rbac.groups` instead.
+  rules: []
+  # -- Deprecated. Use `rbac.groups` instead.
+  staticGroupName: ""
+  # -- Use this to create additional RBAC resources for specified groups.
+  groups:
+    - name: &static-group-name "botkube-plugins-default"
+      rules:
+        - apiGroups: ["*"]
+          resources: ["*"]
+          verbs: ["get", "watch", "list"]
+    - name: &argocd-group "argocd"
+      rules:
+        - apiGroups: [""]
+          resources: ["configmaps"]
+          verbs: ["get", "update"]
+        - apiGroups: ["argoproj.io"]
+          resources: ["applications"]
```

Comment:
Better to check how this behaves if customer never installed ArgoCD CRDs, and just tries to use Botkube only. Will it throw "resource not found"?

Reply:
Right, actually I'll introduce `enabled` property so that we don't create not used resources 👍